Toric/ConvexAnalysis/PolyhedralCone.lean

```diff
@@ -0,0 +1,42 @@
+/-
+Copyright (c) 2025 Paul Reichert. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Paul Reichert
+-/
+import Mathlib
```

Comment:
Can you `#min_imports` this? This is to speed up CI by avoiding building the docs for all of mathlib

Reply:
Sure! :)